components/rmrk/service/NftUtils.ts

```diff
@@ -278,7 +278,7 @@ class NFTUtils {
 
   public static unwrap(rmrkString: string): any {
```

Comment:
please remove, already implemented in https://github.com/vikiival/minimark/blob/master/src/rmrk/unwrap.ts

Reply:
This function are used in the code. In the NftUtils.convert()